be-traveltrove/src/main/java/com/traveltrove/betraveltrove/business/tour/TourServiceImpl.java

```diff
@@ -33,4 +33,19 @@ public Mono<TourResponseModel> getTourByTourId(String tourId) {
                 .map(EntityModelUtil::toTourResponseModel);
     }
 
+    @Override
+    public Mono<TourResponseModel> addTour(TourResponseModel tourResponseModel) {
+        return null;
+    }
+
+    @Override
+    public Mono<TourResponseModel> updateTour(String tourId, TourResponseModel tourResponseModel) {
+        return null;
+    }
+
+    @Override
+    public Mono<Void> deleteTourByTourId(String tourId) {
+        return null;
+    }
```

Comment:
I guess these are not fully implemented yet?

Reply:
no I had done a pr because Maryia needed some of my parts to be finished. there will be another PR tnt after my tests, including some other modifications

Reply:
no I had done a pr because Maryia needed some of my parts to be finished. there will be another PR tnt after my tests, including some other modifications